floater/test/test_utils.py

```diff
@@ -96,10 +96,11 @@ def test_floats_to_netcdf(tmpdir, mitgcm_float_datadir_csv):
     input_dir = str(mitgcm_float_datadir_csv) + '/'
     output_dir = str(tmpdir) + '/'
 
+    # command line options test
     os.chdir(input_dir)
     utils.floats_to_netcdf(input_dir=input_dir, output_fname='test',
                            float_file_prefix='float_trajectories',
-                           ref_time=None, step_time=86400,
+                           ref_time='1993-01-01', step_time='86400',
```

Comment:
@nathanieltarshish I am trying to give a input similar to what we will get from the command line, i.e. a **str**.